code/modules/mob/living/simple_animal/friendly/cockroach.dm

```diff
@@ -60,3 +60,152 @@
 	icon_state = "xfloor1"
 	random_icon_states = list("xfloor1", "xfloor2", "xfloor3", "xfloor4", "xfloor5", "xfloor6", "xfloor7")
 	beauty = -300
+
+/mob/living/simple_animal/hostile/fly
+	name = "fly"
+	desc = "We went to space to escape flies, but they always seem to find us."
+	icon_state = "fly-10"
+	icon_living = "fly-10"
+	icon_dead = "fly_dead"
+	icon = 'icons/mob/bees.dmi'
+	gender = NEUTER
+	speak_emote = list("buzzes")
+	emote_hear = list("buzzes")
+	turns_per_move = 0
+	melee_damage_lower = 0
+	melee_damage_upper = 0
+	retreat_distance = 2
+	minimum_distance = 2
+	friendly = "annoys"
+	attacktext = "seriously annoys"
+	response_help  = "shoos"
+	response_disarm = "swats away"
+	response_harm   = "squashes"
+	maxHealth = 1
+	health = 1
+	spacewalk = TRUE
+	faction = list("hostile")
+	move_to_delay = 0
+	obj_damage = 0
+	environment_smash = ENVIRONMENT_SMASH_NONE
+	mouse_opacity = MOUSE_OPACITY_OPAQUE
+	pass_flags = PASSTABLE | PASSGRILLE | PASSMOB
+	density = FALSE
+	mob_size = MOB_SIZE_TINY
+	mob_biotypes = list(MOB_ORGANIC, MOB_BUG)
+	movement_type = FLYING
+	gold_core_spawnable = HOSTILE_SPAWN
+	search_objects = 1 //have to find those plant trays!
+
+/mob/living/simple_animal/hostile/fly/Initialize()
+	. = ..()
+	AddComponent(/datum/component/swarming)
```

Comment:
AddComponent(/datum/component/beauty, -`amount of ur choosing`)

Reply:
thaaaaaaaaank uuuuuuuu